src/core_test/mpas_test_core.F

```diff
@@ -138,7 +138,7 @@ function test_core_run(domain) result(iErr)!{{{
       end if
 
       !$omp parallel default(firstprivate) shared(domain, threadErrs)
-      call test_core_test_fields(domain, threadErrs, ierr)
+      call test_core_test_fields(domain, threadErrs, iErr)
```

Comment:
I'd generally agree that consistency (of capitalization, in this case) is great, but since this is the only modification to this file, I'd vote for not making this change and having one fewer file in the list of modified files.

Reply:
I would agree with you that a pedantic fix like that normally wouldn't be something to add to a commit like this, but as far as I can tell, if I leave it as "ierr", lines 142-146 won't actually report test failures caused by the field tests, as they check the "iErr" variable, not "ierr".

Reply:
Fortran isn't case sensitive, so I'm surprised that changing between `ierr` and `iErr` makes a difference.